Tests/Realm.Tests/Sync/SessionTests.cs

```diff
@@ -56,22 +56,22 @@ public class SessionTests : SyncTestBase
         [Preserve]
         static SessionTests()
         {
-            var preserveRecoverHandler = new RecoverUnsyncedChangesHandler
+            _ = new RecoverUnsyncedChangesHandler
```

Comment:
What is this test doing?

Reply:
Not sure - I think it may have been added because we create some things with reflection, but it's possible it's no longer needed. Can't check now, so will merge as is, but will keep it in mind to remove at some point.